sql/core/src/test/scala/org/apache/spark/sql/jdbc/JDBCV2Suite.scala

```diff
@@ -2673,4 +2673,23 @@ class JDBCV2Suite extends QueryTest with SharedSparkSession with ExplainSuiteHel
     val indexes3 = jdbcTable.listIndexes()
     assert(indexes3.isEmpty)
   }
+
+  test("IDENTIFIER_TOO_MANY_NAME_PARTS: " +
```

Comment:
Does the test improve the test coverage additionally to QueryCompilationErrorsSuite.scala?

Reply:
> Does the test improve the test coverage additionally to QueryCompilationErrorsSuite.scala?

Yep, this test case tests `IdentifierHelper.asFunctionIdentifier` function, but the test cases in the `QueryCompilationErrorsSuite ` test `MultipartIdentifierHelper.asTableIdentifier` function only.